src/status_im/signing/eip1559.cljs

```diff
@@ -0,0 +1,64 @@
+(ns status-im.signing.eip1559
+  (:require [re-frame.core :as re-frame]
+            [status-im.ethereum.json-rpc :as json-rpc]
+            [status-im.utils.config :as config]
+            [status-im.utils.money :as money]))
+
+(def activation-blocks
+  {"123" (money/bignumber 0)})
```

Comment:
whats that ?

Reply:
it's the chain id and the block that london will be enabled on:
Say 
```
{"1" 200008}
```
Means that on mainnet it will be activated on block 200008 
I can add a comment to clarify